Cargo.toml

```diff
@@ -26,7 +26,7 @@ num-traits = "0.2"
 reqwest = { version = "0.12", default-features = false, features = [
   "rustls-tls-native-roots",
 ] }
-review-database = { git = "https://github.com/petabi/review-database.git", tag = "0.29.1" }
+review-database = { git = "https://github.com/petabi/review-database.git", rev = "c3343e3" }
```

Comment:
This commit includes changes that are unrelated to the current PR. While it can be beneficial to use the latest commit if you’re familiar with its contents, it’s generally better to use the closest relevant commit that directly supports the PR. However, when releasing a new version, we rely on the latest releases from other modules.

Reply:
I understood what you said. I have just replaced it with a commit hash that is directly related to this PR and force-pushed it.